_posts/2015-05-30-hot-planet-astrobiology.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: Global warming may be key to finding E.T.
+author: neuroamanda
+---
+
+_Exoplanets that are too hot may be a sign of alien life_
+
+A new research paper suggests that scanning for waste heat radiating off of Earth-like planets may be important in the search for extraterrestrial intelligent life.
+Like the radio signals emitted by Earth, heat may serve as an interstellar smoke signal indicating energy production and consumption.
+While this is not a new idea, it may be a consideration for how we build future telescopes.
+
+With nearly 2000 exoplanets discovered [so far](http://planetquest.jpl.nasa.gov/newworldsatlas) &mdash; of which 40-odd are [potentially habitable](https://en.wikipedia.org/wiki/List_of_potentially_habitable_exoplanets) &mdash; Earth may soon no longer be unique in its ability to harbor life.
```

Comment:
> "…Earth may soon no longer be unique in its ability to harbor life."

There is a certain universality to this statement that I don't like. "… Earth may soon no longer be the only planet we know of that has the ability to harbor life."